src/Test/QuickCheck/Laws/Data/HeytingAlgebra.purs

```diff
@@ -90,7 +90,7 @@ checkHeytingAlgebra _ = do
   absorption op1 op2 a b = (a `op1` (a `op2` b)) == a
 
   idempotent ∷ (a → a → a) → a → a → Boolean
-  idempotent op a b = a `op` a == a
+  idempotent op a _ = a `op` a == a
```

Comment:
I'm pretty sure the real fix here is to not include a second `b` argument of type `a`.

Reply:
@garyb Do you have thoughts on this? I'm going to go ahead and merge to make the warnings release, but please let us know if this is incorrect!